responsibilities.md

```diff
@@ -45,3 +45,18 @@ The processes needed to support the above responsibilities are still under devel
 ### Board reporting
 
  - It is the responsibility of the Chair to provide regular and timely reports for the Governing Board
+
+### Project reporting
```

Comment:
We would start with projects, and we would expand it to SIGs and Standards in a later batch

Reply:
Agreed. Added.